datalad/core/local/diff.py

```diff
@@ -93,7 +93,7 @@ class Diff(Interface):
             metavar="REVISION",
             doc="""state to compare against the original state, as given by
             any identifier that Git understands. If none is specified,
-            the state of the worktree will be used compared.""",
+            the state of the working tree will be compared.""",
```

Comment:
When fixing this typo, it bothered me a bit that the description isn't quite accurate.  This command does more than compare the files in the working tree to `fr`.  There's also an index comparison that comes from the `git ls-files --modified [...]` call.  As a result, a file will show up as modified even if there are no differences between the working tree and `fr`, but there are differences between the file in the working tree and index.

I couldn't think of clear/concise wording for this, though, and it doesn't really matter in practice given DataLad doesn't emphasize/expose the index.